keel-sql/src/test/kotlin/com/netflix/spinnaker/keel/sql/spring/SpringStartupTests.kt

```diff
@@ -28,6 +29,7 @@ import strikt.assertions.isA
     "spring.datasource.driver-class-name=org.testcontainers.jdbc.ContainerDatabaseDriver"
   ]
 )
+@DisableSpringScheduling
 internal class SpringStartupTests {
```

Comment:
Scheduling was enabled here and I was seeing a bunch of noise from calls to mocked repositories